circus-train-comparator/src/test/java/com/hotels/bdp/circustrain/comparator/hive/functions/PathToPathMetadataTest.java

```diff
@@ -38,15 +38,14 @@
 import org.mockito.Mock;
 import org.mockito.runners.MockitoJUnitRunner;
 
-import com.hotels.bdp.circustrain.comparator.hive.functions.PathToPathMetadata;
 import com.hotels.bdp.circustrain.comparator.hive.wrappers.PathMetadata;
 
 @RunWith(MockitoJUnitRunner.class)
 public class PathToPathMetadataTest {
 
   private static final String DIR_PATH = "file:/abc";
   private static final String FILE_PATH = DIR_PATH + "/child.dat";
-  private static final long LAST_MODIFIED_DIR = 123456789L;
+  private static final long LAST_MODIFIED_DIR = 0;
```

Comment:
This isn't sufficient to test your change, you probably need to keep this returning some non-zero value but in the assertions testing a folder below check that the last modification time is zero.